src/main/kotlin/me/clip/voteparty/util/UpdateChecker.kt

```diff
@@ -1,16 +1,12 @@
 package me.clip.voteparty.util
 
-import com.google.gson.reflect.TypeToken
-import me.clip.voteparty.VoteParty
 import org.bukkit.plugin.Plugin
 import java.net.URL
 
 object UpdateChecker
 {
 	
-	private const val API = "https://api.spiget.org/v2/resources/%d/versions?size=1&sort=-releaseDate"
-	private val LIST_TYPE = object : TypeToken<List<Version>>()
-	{}.type
+	private const val API = "https://api.spigotmc.org/legacy/update.php?resource=987"
```

Comment:
you removed the format variable `%d` making this hard coded for 987.
(not an issue, just a behaviour change you should be aware of considering the front end hasnt changed)

Reply:
Unhardcoded it.